Разметка внутренней страницы

TimonP opened the pull request:
Добавлена разметка для главной и внутренней страницы

---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/2009235/tasks/3)

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/2009235/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2009235-sedona-33/2/

TimonP committed:
Добавление разметки

(до просмотра второго лайва)

TimonP committed:
разметка

TimonP committed:
разметка

2:18:18(life stop)

TimonP committed:
разметка после просмотра лайва№2

TimonP committed:
закомментированный код переезжает вниз

TimonP committed:
исправление формы

TimonP committed:
разметка

TimonP committed:
Update index.html

убран повторяющийся тег <html lang="ru">

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
+					<button type="button">Хочу сюда!</button>
+					<!-- <a class="buttom_brown" href="catalog.html">Хочу сюда!</a> -->
+				</li>
+			<ul>
```
Ошибка. Вместо закрывающего тега `</ul>` - открывающий

TimonP replied:
Тег теперь закрыт

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
+<head lang="ru">
```
Тут должен быть открывающий тег html

TimonP replied:
Исправлено

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
+
+		<section class="contacts">
+			<address>
+				<h2 class="visually-hidden">Контакты</h2>
```
h2 нельзя вкладывать в address

TimonP replied:
теперь заголовок перенесен из тега address в тег section
<h2 class="visually-hidden">Контакты</h2>

TimonP replied:
`<section class="contacts">
        <h2 class="visually-hidden">Контакты</h2>
			<address>`

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
+		<section class="contacts">
+			<address>
+				<h2 class="visually-hidden">Контакты</h2>
+				<a class="contacts_phone" href="tel:+78128121212">+7 (812) 812-12-12<a>
```
Тег `a` неправильно закрыт

TimonP replied:
теперь тег правильно закрыт)

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
+		<img class="visially-hidden" alt="Закрыть">
+	</button>
+	<form action="action="https://echo.htmlacademy.ru/"" method="post">
+		<p>
+			<label for="start">Дата заезда:</label>
+			<input type="date" value="2022-01-22" id="start" min="2018-07-22" name="datein">
+			Мы не можем отправить вас в прошлое.
+		</p>
+		<p>
+			<label for="finish">Дата выезда:</label>
+			<input type="date" value="2022-01-23" id="finish" name="dateout">
+			На эти даты есть свободные номера. Пока есть.
+		</p>
+		<p>
+			<label>Взрослые:
+			<input type="number" value="1" min="1">
+			</label>
+			((((((второй вариант))))))
+			<div class="count">
+      			<lable for="peple_count">Взрослые:</ladle>
+    			<button class="back" type="button">
+    				<span class="visually-hidden">Уменьшить на одного</span>
+    			</button>
+				<input type="text" id="peple_count" value="1">
+    			<button class="next" type="button">
+    				<span class="visually-hidden">Увеличить на одного</span>
+    			</button>
+			</div>
+
+			<label for="children_count">Дети:</label>
+			<button class="dop_info" type="" alt="Дополнительная информация">i</button>
+			<div class="dop_info_bloc">
+				<p>Укажите количество детей, которые будут с вами, возраст которых от 6 до 18 лет.
+					 Дети до 6 лет размещаются бесплатно.</p>
+			</div>
+			<input type="number" value="0" min="0" id="children_count">
+		</p>
+		<button type="submit">Найти</button>
+	</form>
+	</div> -->
```
Закомментированного кода в коммитах не должно быть

TimonP replied:
этот закоментированный код убран

однако остался во втором файле

не понятно почему для ввода максимальной и минимальных цен надо использовать тип формы text, ведь по сути пользователь должен вводить числовой тип данных

TimonP replied:
во втором файле catalog.html 
`<!--<input type="number" min="0" max="1000000" name="min_prise" placeholder="0">-->`

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
+    <header class="main-header">
+	 <!-- Шапка сайта -->
+	 	<nav class="navigation">
+			<a class="logo" href="https://htmlacademy.ru/">
```
Здесь вообще не должно быть атрибута href, логотип на внутренних страницах ведет на главную, а на главной никуда не должен вести

TimonP replied:
`href="https://htmlacademy.ru/`
удален

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
+		<section class="first_screen1">
+		<!-- Первый блок с логотипами (разделен на два блока) -->
+			<h1 class="visually-hidden">Sedona</h1>
+			<div class="first_part">
+				<img src="" alt="Welcome">
+				<img src="" alt="To the gargeous">
+				<img src="" alt="SEDONA. Because the Grand Canyon sucks!">
+			</div>
```
Это одно сплошное изображение

TimonP replied:
Хорошо, так даже будет проще)

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
+			<p class="h2_p">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+		</section>
+		
+		<section class="benefits">
```
У секции нет заголовка

TimonP replied:
`<h2 class="visually-hidden">Преимущества</h2>`
заголовок добавлен

Muhammadabumansur commented on index.html:
```diff
@@ -1,12 +1,222 @@
-  </body>
+		<section class="search">
+		<!-- Поиск гостиницы -->
+			<h2>Красивая дорога</h2>
```
Не тот текст

TimonP replied:
Действительно)
исправила

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+					<button type="button">Хочу сюда!</button>
+                    <!-- <a class="buttom_brown" href="catalog.html">Хочу сюда!</a> -->
+				</li>
+			<ul>
```
Ошибка. Вместо закрывающего тега `</ul>` - открывающий

TimonP replied:
исправлено

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+            <ul class="crumbs">
+                <li class="crumbs_item">
+                    <a class="crumbs_link" href="index.html">
+                        <img href="" alt="Домой">
```
href вместо src

TimonP replied:
иправлено

TimonP replied:
* исправлено

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+            </ul>             
+        </header>
+
+        <section class="filter">
```
Нет заголовка у секции

TimonP replied:
заголовок добавлен 
`<h2 class="visually-hidden">Каталог гостиниц</h2>`

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+        <section class="first_screen2">
+        <!-- Сортировка -->
+            <h2 class="result-stats">Найдено гостиниц: 38</h2>
+            <ladle>
+                <select class="select-control" aria-label="Сортировка">
+                    <option value="popular">Сначала популярное</option>
+                    <option value="cheap" selected>Сначала дешёвое</option>
+                    <option value="expensive">Сначала дорогое</option>
+                  </select>
+            </ladle>
```
Нет такого элемента ladle

TimonP replied:
Извините за опечатку
ladle исправлен на lable

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+                    <p>Гостиница</p>
+                    <p class="hotel_price">От 2000₽</p>
+                    <a class="left-card-button button" href="#">Подробнее</a>
+                    <button type="button added" >В ибранном</button>
```
Неправильное значение атрибута type

TimonP replied:
исправлено на 
`<button type="button" class="added">В ибранном</button>`

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+                    <a class="pagination-link" href="#">10</a>
+                </li>
+            </ul>
+            <ladle> Гостиниц на странице:
+                <select class="list_count" aria-label="Колличество отображаемых гостинниц">
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4" selected>4</option>
+                  </select>
+            </ladle>         
```
Опять какой-то ladle

TimonP replied:
ladle сменил имя на lable
 теперь он не такой

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+            </div>
+        </section>
+
+        <section class="pagin">
```
Секция без заголовка

TimonP replied:
теперь секция с заголовком 
`<h2 class="visually-hidden">Выборка страниц</h2>`

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+    
+        <section class="contacts">
+            <address>
+                <h2 class="visually-hidden">Контакты</h2>
```
h2 нельзя вкладывать в address

TimonP replied:
H2  переехал
`<section class="contacts">
            <h2 class="visually-hidden">Контакты</h2>
                        <address>`

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+        <section class="contacts">
+            <address>
+                <h2 class="visually-hidden">Контакты</h2>
+                <a class="contacts_phone" href="tel:+78128121212">+7 (812) 812-12-12<a>
```
Тег `a` неправильно закрыт

TimonP replied:
теперь тег закрыт правильно </a>

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+	
+    <main class="main_inner">
+    <!-- Основное содержимое страницы -->
+        <header class="main_header">
```
![image](https://user-images.githubusercontent.com/16342548/151234100-49384a0a-c6d2-44b2-93a2-e924e71fb7e7.png)

Это должен быть один блок. Ему же надо как-то фон задать

TimonP replied:
теперь все завернуто `<header class="header_two">`

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+        <section class="filter">
+        <!-- Фильтр -->
+            <form action="https://echo.htmlacademy.ru/" method="post">
+                <div class="list">
```
Все эти дивы обертки над fieldset не нужны

TimonP replied:
дивы удалены

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+            <form action="https://echo.htmlacademy.ru/" method="post">
+                <div class="list">
+                    <fieldset class="filter_list">
+                        <legend>Инфраструктура:</legend>
+                        <label>
+                            <input type="checkbox" name="swimming_pool" checked> Бассейн
+                        </label>
+                            <br>
+                        <label>
+                            <input type="checkbox" name="Parking" checked> Парковка
+                        </label>
+                            <br>
+                        <label>
+                            <input type="checkbox" name="wi-fi" > Wi-fi
+                        </label>
+                    </fieldset>
+                </div>
+                    
+                <div class="list">   
+                    <fieldset class="filter_list">
+                        <legend>Тип жилья:</legend>
+                        <label>
+                            <input type="radio" name="roomtype" value="Hotel" checked> Гостинница
+                        </label>
+                            <br>
+                        <label>
+                            <input type="radio" name="roomtype" value="Motel"> Мотель
+                        </label>
+                            <br>
+                        <label>
+                            <input type="radio" name="roomtype" value="Appartaments"> Апартаменты
+                        </label>
```
Все br здесь нужно удалить

TimonP replied:
br удален

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+                <div class="prise">
+                    <fieldset class="filter_list2">
+                        <legend>Стоимость, <span>₽</span></legend>
+                        <label>
+                            <input type="text" name="min_prise" placeholder="0">
+                            <!--<input type="number" min="0" max="1000000" name="min_prise" placeholder="0">-->
+                            от
+                        </label>
+                        <label>
+                            <input type="text" name="max_prise" placeholder="9000">
+                            <!--<input type="number" min="0" max="1000000" name="max_prise" placeholder="9000">-->
+                            до
+                        </label>
```
Эти два label нужно объединить в один блок

TimonP replied:
точно, для последующей стилизации
инкапсулировано в блок <div>

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+                            до
+                        </label>
+                        <br>
+                        <label>
+                            <input type="range" id="volume" min="0" max="1000000" value="9000" step="1">
+                        </label>
```
Тут не нужен range, range с двумя ползунками в html нет

TimonP replied:
Спасибо за информацию, я учту что тут неверно.

данный тип был добавлен исходя из информации во втором лайве
2:10:50 https://www.youtube.com/watch?v=v47bHil7wAg&t=7857s

Предлагаю оставить эти строки как есть и до выхода обучающего материала не обращать на них внимание.

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+                    <option value="expensive">Сначала дорогое</option>
+                  </select>
+            </ladle>
+            <a href="#" class="button button-light">
+                <span class="visually-hidden">Показать карточки</span>
+            </a>
+            <a class="button" href="#">
+                <span class="visually-hidden">Показать по 1</span>
+            </a>
+            <a class="button" href="#">
+                <span class="visually-hidden">Показать список</span>
+             </a>
```
Эти ссылки должны идти списком

TimonP replied:
исправлено

`<ul class="visually-result">
                <li>
                    <a href="#" class="button button-light">
                        <span class="visually-hidden">Показать карточки</span>
                    </a>
                </li>
                <li>
                    <a href="#" class="button" >
                        <span class="visually-hidden">Показать по 1</span>
                    </a>
                </li>
                <li>
                    <a href="#" class="button">
                        <span class="visually-hidden">Показать список</span>
                    </a>
                </li>
            </ul>`

Muhammadabumansur commented on catalog.html:
```diff
@@ -0,0 +1,307 @@
+                    <p class="hotel_price">От 4000₽</p>
+                    <a class="left-card-button button" href="#">Подробнее</a>
+                    <button type="button">В ибранное</button>
+                    <p>
+                        <img class="star" src="" alt="&Star;">
+                        <img class="star" src="" alt="&Star;">
+                        <img class="star" src="" alt="&Star;">
+                        <img class="star" src="" alt="&Star;">
+                        <span class="rate">Рейтинг: 8,5</span>
+                    </p>
```
Эти звездочки надо сделать фоном для одного элемента а рейтинг это отдельный элемент. Такой alt не стоит делать, лучше элементу задать aria-label типа "4 звезды" "5 звезд" чтоб было информативно

TimonP replied:
Не знаю насколько правилен такой вариант.
беграунд в тогда думаю повесить на span с class="two_star" 
второму span class="rate" прописать свои в стилях  float: right
     `					<span class="visually-hidden">2 звезды</span>
					<span class="two_star"></span>
					<span class="rate">Рейтинг: 9,2</span>	`

TimonP replied:
`                    <div class="two_star">
                        <p class="visually-hidden">2 звезды</p>
                    </div>
                    <div class="rate">
                        <p>Рейтинг: 9,2</p>
                    </div>   `

Muhammadabumansur commented:
Очень много невалидного кода, проверяй код в валидаторе https://validator.w3.org/nu/#textarea это удобно
Код не выровнен с помощью отступов, очень трудно читать.

Нужно исправить все замечания

TimonP committed:
Исправления разметки

TimonP committed:
замена пробелов на табуляцию

TimonP committed:
убраны лишние пробелы

TimonP committed:
замена тега p на span

					<div class="two_star">
						<p class="visually-hidden">2 звезды</p>
						<span class="visually-hidden">2 звезды</span>
					</div>
					<div class="rate">
						<p>Рейтинг: 9,2</p>
						<span>Рейтинг: 9,2</span>
					</div>

TimonP committed:
div для звезд и рейтинга убраны

TimonP committed:
разметка
